src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -259,14 +267,6 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
 		handle_message_vision_position_estimate(msg);
 		break;
 
-	case MAVLINK_MSG_ID_ATTITUDE_QUATERNION_COV:
```

Comment:
who was using MAVLINK_MSG_ID_ATTITUDE_QUATERNION_COV and MAVLINK_MSG_ID_LOCAL_POSITION_NED_COV? What are we breaking?

Reply:
The two msgs I am aware they are here because there was no msg that could be used to parse the covariance together with the estimator data. But actually those msgs should be used as a feedback, just as `LOCAL_POSITION_NED`. If it actually breaks any setup, I am not sure. But the current setup was actually an alternative, since there was no msg structure to carry both pose, velocity and covariances. I actually took off these msgs cause it was overflowing fmu-v2.